HealthItem/src/test/java/com/github/sef24sp4/healthitem/HealthItemTest.java

```diff
@@ -0,0 +1,62 @@
+package com.github.sef24sp4.healthitem;
+
+import com.github.sef24sp4.common.entities.ICollidableEntity;
+import com.github.sef24sp4.common.interfaces.IEntityManager;
+import com.github.sef24sp4.common.item.ItemRarity;
+import com.github.sef24sp4.common.metadata.GameElementType;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+class HealthItemTest {
+	@Mock
+	private IEntityManager mockEntityManager = mock(IEntityManager.class);
+
+	@Mock
+	private ICollidableEntity mockEntity = mock(ICollidableEntity.class);
+
+	@Mock
+	private HealthItem healthItem = new HealthItem();
+
+	@Mock
+	private HealthItemProvider healthItemProvider = new HealthItemProvider();
+
+	@Test
+	void collide() {
+		//Setup
+		this.mockEntityManager.addEntity(this.healthItem);
+		when(this.mockEntity.getType()).thenReturn(GameElementType.PLAYER);
+		//Method call
+		this.healthItem.collide(this.mockEntityManager, this.mockEntity);
+		//Verification
+		verify(this.mockEntityManager).removeEntity(this.healthItem);
+	}
+
+	@Test
+	void collideNotPlayer() {
+	    //Setup
+	    this.mockEntityManager.addEntity(this.healthItem);
+		when(this.mockEntity.getType()).thenReturn(GameElementType.ENEMY);
+		//Method call
+		this.healthItem.collide(this.mockEntityManager, this.mockEntity);
```

Comment:
Please fix indentation

Reply:
Not quiet sure whats wrong with the marked lines?

Reply:
These lines are indented one level too deep

Reply:
I can see the indentation on github, but not when accessing the file from a file editor lol? Thought it mby was a problem with IntelliJ, but I even tried with notepad :/

Reply:
Try in Intellij to reformat the whole file.